src/Statement/StatementGuid.php

```diff
@@ -55,6 +61,13 @@ public function getEntityId() {
 		return $this->entityId;
 	}
 
+	/**
+	 * @return string
```

Comment:
You can ditch the comment in favor of a return type hint.

Reply:
Happy to, but looks like there aren't any return type hints in this repo yet? (or at least not in the few files I looked at)